awscli/topics/config-vars.rst

```diff
@@ -476,6 +478,38 @@ To set encoding used for text files different from the locale, you can use
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+Pager
+-----
+
+AWS CLI uses pager for output data that don't fit on the screen.
+
+By default if on Linux/MacOS, use ``less`` as the pager on Windows
```

Comment:
Let's reword this to: "If on Linux/MacOS, `less` is used as the default pager. On Windows, the default is `more`."